### ISSUE TITLE
Support for Firefox policies.json

### ISSUE THREAD

**Issue by pujvz** [einaregilsson/Redirector#334]
It would be great to have support for deploying Redirector settings in Firefox using policies.json -
https://github.com/mozilla/policy-templates/blob/master/README.md#3rdparty
https://extensionworkshop.com/documentation/enterprise/adding-policy-support-to-your-extension/

I believe this would be a useful feature. Redirector already has the ability to export redirects. Being able to add these exported redirects to a policies.json and deploy it to many computers would be extremely helpful.

**Comment by Gitoffthelawn** [comment]
I agree that this will be useful to support, and could save people quite a bit of time/energy.

If you, or someone else, would like to write the code to implement it, I can get it integrated and released.

**Comment by sewnie** [comment]
if it is considered useful, how come it still was not merged and just ignored?
